geograypher/meshes/meshes.py

```diff
@@ -585,7 +581,7 @@ def create_pytorch3d_mesh(
             textures=texture,
         ).to(self.device)
 
-        if batch_size != 1 and len(self.pytorch3d_mesh) == 1:
+        if batch_size != len(self.pytorch3d_mesh):
```

Comment:
I was thinking that this method would return the mesh and we'd entirely remove the `self.pytorch3d_mesh` attribute. Then, this line would want to check if batch size is not 1.

Reply:
I am still confused about why we would check if the batch_size is not 1 and the length of the mesh is 1. Would it not be more intuitive to check if both values are the same since that is what the rasterizer method requires?

Reply:
I see your point. But we know the mesh will always be length 1 when we create it. I left it as is.